tools/cli/go-whisk-cli/commands/commands.go

```diff
@@ -30,6 +30,7 @@ import (
 
 var client *whisk.Client
 const DefaultOpenWhiskApiPath string = "/api"
+var UserAgent string = "OpenWhisk-CLI"
```

Comment:
Looks like this should be a const.

Reply:
not a const so that it can be overwritten by the plugin etc